src/components/ftbot/PairlistConfigurator.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <b-container fluid>
+    <b-row align-v="stretch">
+      <b-col cols="12" md="3" style="height: 760px; overflow-y: scroll">
```

Comment:
We shouldn't use fixed heights (and for sure not like this).

It looks quite odd even on my screen, as it stops ~5cm above the lower end of the screen.

Reply:
Ah yes, I put it in temporarily, but want to make it so it'll use what vertical space is available.